src/Products/Signature/Controllers/SignatureApiController.cs

```diff
@@ -1093,27 +1119,28 @@ private void AddSignOptions(string documentType, SignatureOptionsCollection sign
         /// <param name="password">string</param>
         /// <param name="signsCollection">SignatureOptionsCollection</param>
         /// <returns></returns>
-        private SignedDocumentEntity SignDocument(string documentGuid, string password, SignatureOptionsCollection signsCollection)
+        private SignedDocumentEntity SignDocument(string documentGuid, string password, List<SignOptions> signsCollection)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make 'SignDocument' a static method.](https://app.codacy.com/gh/groupdocs-signature/GroupDocs.Signature-for-.NET-MVC/pullRequest?prid=5325146)